test/syncenginetestutils.h

```diff
@@ -85,6 +90,14 @@ class DiskFileModifier : public FileModifier
     void mkdir(const QString &relativePath) override {
         _rootDir.mkpath(relativePath);
     }
+    void move(const QString &relativePath, const QString &relativeDestinationDirectory) override {
+        auto orig = _rootDir.filePath(relativePath);
+        auto dest = _rootDir.filePath(relativeDestinationDirectory);
+        QVERIFY(QFileInfo::exists(orig));
+        QVERIFY(QFileInfo::exists(dest));
+        QVERIFY(QFileInfo(dest).isDir());
+        QVERIFY(_rootDir.rename(relativePath, relativeDestinationDirectory));
```

Comment:
What I mean is that there is nothing in the rename documentation that you can achieve "A/a1" --> "B/a1" by doing `rename("A/a1", "B/")`. You want to move a1 into B, and to do that wouldn't the correct way to do `rename("A/a1", "B/a1")`?